### PR TITLE
[checkbox-ce-oem] Add moredetail for ce-oem-mtd/chec-total-numbers (Bugfix)

### PR DESCRIPTION
**Pull request by LiaoU3**
## Description
Add more details for the job, so it will be easy to debug.

## Resolved issues
Add more details

## Documentation
N/A

## Tests
When not setting TOTAL_MTD_NUM
```terminal
===========================[ Running Selected Jobs ]============================
==============[ Running job 1 / 1. Estimated time left: 0:00:05 ]===============
----------------[ Check the total MTD numbers for the platform ]----------------
ID: com.canonical.contrib::ce-oem-mtd/check-total-numbers
Category: com.canonical.contrib::mtd
... 8< -------------------------------------------------------------------------
TOTAL_MTD_NUM is not defined!
------------------------------------------------------------------------- >8 ---
Outcome: job failed
Finalizing session that hasn't been submitted anywhere: checkbox-run-2024-05-31T06.56.32
==================================[ Results ]===================================
 ☒ : Check the total MTD numbers for the platform
```
When setting the wrong TOTAL_MTD_NUM
```
===========================[ Running Selected Jobs ]============================
==============[ Running job 1 / 1. Estimated time left: 0:00:05 ]===============
----------------[ Check the total MTD numbers for the platform ]----------------
ID: com.canonical.contrib::ce-oem-mtd/check-total-numbers
Category: com.canonical.contrib::mtd
... 8< -------------------------------------------------------------------------
The number of MTD is incorrect!
Expected: 5
Actual: 17
------------------------------------------------------------------------- >8 ---
Outcome: job failed
Finalizing session that hasn't been submitted anywhere: checkbox-run-2024-05-31T07.02.07
==================================[ Results ]===================================
 ☒ : Check the total MTD numbers for the platform
```
When setting the right TOTAL_MTD_NUM
```
===========================[ Running Selected Jobs ]============================
==============[ Running job 1 / 1. Estimated time left: 0:00:05 ]===============
----------------[ Check the total MTD numbers for the platform ]----------------
ID: com.canonical.contrib::ce-oem-mtd/check-total-numbers
Category: com.canonical.contrib::mtd
... 8< -------------------------------------------------------------------------
The number of MTD is correct!
Expected: 17
Actual: 17
------------------------------------------------------------------------- >8 ---
Outcome: job passed
Finalizing session that hasn't been submitted anywhere: checkbox-run-2024-05-31T07.05.47
==================================[ Results ]===================================
 ☑ : Check the total MTD numbers for the platform
```

### CODE REVIEW

File: contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mtd/jobs.pxu [hunk 1]
```diff
@@ -9,7 +9,13 @@ plugin: shell
 user: root
 flags: also-after-suspend
 estimated_duration: 5
```
Comment (rickwu666666) [inline contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mtd/jobs.pxu:11]:
```suggestion
estimated_duration: 5
requires: manifest.has_mtd == 'True'
imports: from com.canonical.plainbox import manifest
```

Comment (rickwu666666) [reply contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mtd/jobs.pxu:11]: I suggest that with use manifest in this job as well.

**Review by rickwu666666 — CHANGES_REQUESTED** [review]

File: contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mtd/jobs.pxu (hunk 1 above)
Comment (LiaoU3) [reply contrib/checkbox-ce-oem/checkbox-provider-ce-oem/units/mtd/jobs.pxu:11]: Great idea!

**Review by baconYao — APPROVED** [review]
+1

**Review by rickwu666666 — APPROVED** [review]
LGTM

**Pull request merged**